.vscode/settings.json

```
@@ -8,7 +8,9 @@
   "typescript.tsserver.experimental.enableProjectDiagnostics": false,
   "typescript.tsdk": "node_modules/typescript/lib",
   "files.autoSaveWhenNoErrors": true,
```

Biome (2.4.6)

[error] The key files.autoSaveWhenNoErrors was already declared.

(lint/suspicious/noDuplicateObjectKeys)